<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>49</y>
      <width>681</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="MaingridLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>打开文件</string>
    </property>
    <addaction name="fileopen"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionShow"/>
   <addaction name="actionClose"/>
  </widget>
  <action name="fileopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/r.png</normaloff>:/img/r.png</iconset>
   </property>
   <property name="text">
    <string>show</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/r.png</normaloff>:/img/r.png</iconset>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
